packages/astro/src/@types/astro.ts

```diff
@@ -1473,15 +1473,15 @@ export interface AstroUserConfig {
 			 * @docs
 			 * @kind h4
 			 * @name experimental.i18n.locales
-			 * @type {string[]}
+			 * @type {Locales}
 			 * @version 3.5.0
 			 * @description
 			 *
 			 * A list of all locales supported by the website (e.g. `['en', 'es', 'pt-br']`). This list should also include the `defaultLocale`. This is a required field.
 			 *
 			 * No particular language format or syntax is enforced, but your folder structure must match exactly the locales in the list.
```

Comment:
```suggestion
			 * Languages can be listed either as individual codes (e.g. `['en', 'es', 'pt-br']`) or mapped to a shared `path` of codes (e.g.  `{ path: "english", codes: ["en", "en-US"]}`). These codes will be used to determine the URL structure of your deployed site.
			 * 
			 * No particular language code format or syntax is enforced, but your project folders containing your content files must match exactly the locales in the list. In the case of multiple `codes` pointing to a custom URL path prefix, store your content files in a folder that exactly matches the first code listed. 
```
As usual, not sure whether this is true or necessary, but it's explicit enough to provide clear guidance!  :sweat_smile: 

I would say we're probably at the point we need about this level of detail/advice here, so whatever is actually true, let's try to end up with something kind of like this?

Reply:
> In the case of multiple `codes` pointing to a custom URL path prefix, store your content files in a folder that exactly matches the first code listed.

This should be

> In the case of multiple `codes` pointing to a custom URL path prefix, store your content files in a folder that exactly matches the `path`.

The `codes` come into play only when we want to check the **preferred locale(s)** from the browser when we read the `Accept-Language` header.

Reply:
Sounds good! I updated the example.